Manito/Manito/Global/Extension/Date+Extension.swift

```diff
@@ -19,4 +19,34 @@ extension Date {
         formatter.dateFormat = "yyyy.MM.dd"
         return formatter.string(from: self)
     }
+    
+    func isToday() -> Bool {
+        let now = Date()
+        let distance = self.distance(to: now)
+        if distance > 0 && distance < 86400 {
+            return true
+        } else {
+            return false
+        }
```

Comment:
p3;

```suggestion
        return distance > 0 && distance < 86400
```
이렇게 해도 괜찮지 않나용?

Reply:
밑에 38, 46번도 같은 의견입니다!

Reply:
아하 그렇군요 ! 꿀팁 감삼당 ^_______^